src/linters.ts

```diff
@@ -306,6 +306,10 @@ function lintBindingExpression(parsed,
                                implicitBindings,
                                name) {
   var undeclaredBindings = [];
+  properties = properties.concat([{
```

Comment:
If this is a temporary fix please add comment explaining why this is being added & when it will be safe to remove.